R/statistic-histogram.R

```diff
@@ -73,8 +93,16 @@ histogram.getParameters <- function(mechanism, n.bins, n, accuracy, delta=10^-6,
 
 #' Histogram confidence interval
 #' 
-#' Return the confidence interval for the differentially private histogram release given the
-#' accuracy.
+#' Return the confidence interval for each bins of the differentially private histogram
+#' release, given the accuracy.
+#' 
+#' A confidence interval indicates the range in which we estimate the true value of a
+#' statistic to be. In this case, the confidence interval indicates the range in which
+#' the true count for each histogram bin could be. In this histogram statistic, we use
+#' a 95% confidence interval. To give an example: say a differentially private release 
```

Comment:
I'd recommend not saying "in this histogram statistic we use a 95% confidence interval", since statistical significance has a default of 5% but that could be passed in differently by the user and isn't hardcoded into this function. :)

Also, roxygen input states accuracy here is a percentage, and assumes it is such in the calculation of the CI. But in general we aren't calculating accuracy as a percentage of the data range or anything. So we should change that to just have accuracy be passed in as it is calculated and used everywhere else.

Reply:
The fact that it treats the accuracy as a percentage explains A LOT, the CIs have been huge and I have not been sure why.